src/main/java/org/broadinstitute/hellbender/tools/genomicsdb/GenomicsDBImport.java

```diff
@@ -810,10 +810,11 @@ private InitializedQueryWrapper(final FeatureReader<VariantContext> reader, fina
         }
 
         @Override
-        public CloseableTribbleIterator<VariantContext> query(final String chr, final int start, final int end) {
+        public CloseableTribbleIterator<VariantContext> query(final String chr, final int start, final int end) throws IOException {
             final SimpleInterval queryInterval = new SimpleInterval(chr, start, end);
             if( !interval.equals(queryInterval)){
-                throw new GATKException("Cannot call query with different interval, expected:" + this.interval + " queried with: " + queryInterval);
+                this.interval = new SimpleInterval(chr, start, end);
+                this.query = reader.query(interval.getContig(), interval.getStart(), interval.getEnd());
```

Comment:
@droazen @lbergelson would it be possible for one of you take a look at this fix for #5300? I think my fix preserves the original intent of InitializedQueryWrapper which appears to be to speed up the opening of VCF readers. I didn't write this part of the code so I may not fully understand the objective.